compiler/AST/build.cpp

```diff
@@ -2043,30 +2062,28 @@ DefExpr* buildClassDefExpr(const char*  name,
                            BlockStmt*   decls,
                            Flag         isExtern,
                            const char*  docs) {
-  AggregateType* ct = new AggregateType(tag);
+  AggregateType* ct = NULL;
+  TypeSymbol* ts = NULL;
+
+  ct = new AggregateType(tag);
 
   // Hook the string type in the modules
   // to avoid duplication with dtString created in initPrimitiveTypes().
   // gatherWellKnownTypes runs too late to help.
-  if (strcmp("string", name) == 0) {
-    *dtString = *ct;
-
-    // These fields get overwritten with `ct` by the assignment.
-    // These fields are set to `this` by the AggregateType constructor
-    // so they should still be `dtString`. Fix them back up.
-    dtString->fields.parent   = dtString;
-    dtString->inherits.parent = dtString;
-
-    gAggregateTypes.remove(gAggregateTypes.index(ct));
-
-    delete ct;
-
-    ct = dtString;
+  if (strcmp("_string", name) == 0) {
+    gdbShouldBreakHere();
```

Comment:
Should these gdbShouldBreakHere() calls be removed now?  (were they debugging code?)